xkb/xkbtext.c

```diff
@@ -1261,7 +1261,7 @@ XkbActionText(XkbDescPtr xkb, XkbAction *action, unsigned format)
 char *
 XkbBehaviorText(XkbDescPtr xkb, XkbBehavior * behavior, unsigned format)
 {
-    char buf[256];
+    char buf[256] = { 0 };
```

Comment:
Isn't it better to only set the first byte to 0? Not that it really matters if 255 extra bytes are set

Reply:
hmm, just looking purely at the source code, this feels nicer to me.
does it make a practical difference (in actual runtime) when all is zero'ed ?
it seems a pretty cold code path, not called very often.

what's the actually generated machine code looking like ?